tasks/section_03.yml

```diff
@@ -1,19 +1,215 @@
+    - section3.1
+
+- name: 3.2 Ensure that docker.service file permissions are set to 644 or more restrictive (Scored)
+  file:
+    path: /lib/systemd/system/docker.service
+    mode: 0644
+  tags:
+    - scored
+    - section3
+    - section3.2
+
+- name: 3.3 Ensure that docker.socket file ownership is set to root:root (Scored)
+  file:
+    path: /lib/systemd/system/docker.socket
+    state: touch
+    owner: root
+    group: root
+  tags:
+    - scored
+    - section3
+    - section3.3
+
+- name: 3.4 Ensure that docker.socket file permissions are set to 644 or more restrictive (Scored)
+  file:
+    path: /lib/systemd/system/docker.socket
+    mode: 0644
+  tags:
+    - scored
+    - section3
+    - section3.4
+
+- name: 3.5 Ensure that /etc/docker directory ownership is set to root:root (Scored)
+  file:
+    path: /etc/docker
+    state: directory
+    owner: root
+    group: root
+  tags:
+    - scored
+    - section3
+    - section3.5
+
+- name: 3.6 Ensure that /etc/docker directory permissions are set to 755 or more restrictive (Scored)
+  file:
+    path: /etc/docker
+    state: directory
+    mode: 0755
+  tags:
+    - scored
+    - section3
+    - section3.6
+
+- name: 3.7 Ensure that registry certificate file ownership is set to root:root (Scored)
+  file:
+    path: "{{ tls_registry_cert }}"
+    state: touch
+    owner: root
+    group: root
+  tags:
+    - scored
+    - section3
+    - section3.7
+  when: tls_registry_cert is defined
+
+- name: 3.8 Ensure that registry certificate file permissions are set to 444 or more restrictive (Scored)
+  file:
+    path: "{{ tls_registry_cert }}"
+    state: touch
+    mode: 0444
+  tags:
+    - scored
+    - section3
+    - section3.8
+  when: tls_registry_cert is defined
+
+- name: 3.9 Ensure that TLS CA certificate file ownership is set to root:root (Scored)
+  file:
+    path: "{{ tls_ca_cert }}"
+    state: touch
+    owner: root
+    group: root
+  when: tls_ca_cert is defined
+  tags:
+    - scored
+    - section3
+    - section3.9
+
+- name: 3.10 Ensure that TLS CA certificate file permissions are set to 444 or more restrictive (Scored)
+  file:
+    path: "{{ tls_ca_cert }}"
+    state: touch
+    mode: 0444
+  when: tls_ca_cert is defined
+  tags:
+    - scored
+    - section3
+    - section3.10
+
+- name: 3.11 Ensure that Docker server certificate file ownership is set to root:root (Scored)
+  file:
+    path: "{{ tls_cert }}"
+    state: touch
+    owner: root
+    group: root
+  when: tls_cert is defined
+  tags:
+    - scored
+    - section3
+    - section3.11
+
+- name: 3.12 Ensure that Docker server certificate file permissions are set to 444 or more restrictive (Scored)
+  file:
+    path: "{{ tls_cert }}"
+    state: touch
+    mode: 0444
+  when: tls_cert is defined
+  tags:
+    - scored
+    - section3
+    - section3.12
+
+- name: 3.13 Ensure that Docker server certificate key file ownership is set to root:root (Scored)
+  file:
+    path: "{{ tls_key }}"
```

Comment:
If this is intended as a check that whatever setup the node didn't screw up and set the wrong perms, then it's going to silently just fix the problem rather than warning us that whatever did the initial setup is bad and needs fixing?

Reply:
🤔 Good point, though, since this playbook is supposed to set this up in Docker it should be handled somewhere in this role.